pkg/registry/controller/etcd/etcd.go

```diff
@@ -32,8 +32,24 @@ type REST struct {
 }
 
 // NewREST returns a RESTStorage object that will work against replication controllers.
-func NewREST(s storage.Interface) (*REST, *StatusREST) {
+func NewREST(s storage.Interface, useCacher bool) (*REST, *StatusREST) {
 	prefix := "/controllers"
+
+	storageInterface := s
```

Comment:
Just take a runtime.Object, which is what stores that RC anyway.

On Thu, Oct 29, 2015 at 11:23 AM, Wojciech Tyczynski <
notifications@github.com> wrote:

> In pkg/registry/controller/etcd/etcd.go
> <https://github.com/kubernetes/kubernetes/pull/16496#discussion_r43424929>
> :
>
> >  	prefix := "/controllers"
> > +
> > +	storageInterface := s
>
> Can we pass types to a function?
> I'm using:
> "Type: &api.ReplicationController{},"
> below.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/16496/files#r43424929>.
>